src/Dfc.CourseDirectory.WebV2/Features/TLevels/ViewAndEditTLevel/EditTLevelController.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Dfc.CourseDirectory.Core.Models;
+using Dfc.CourseDirectory.WebV2.Filters;
+using FormFlow;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Dfc.CourseDirectory.WebV2.Features.TLevels.ViewAndEditTLevel
+{
+    [Route("t-levels/{tLevelId}")]
+    [JourneyMetadata(
+        journeyName: "EditTLevel",
+        stateType: typeof(EditTLevelJourneyModel),
+        appendUniqueKey: true,
+        requestDataKeys: "tLevelId")]
+    [RequireFeatureFlag(FeatureFlags.TLevels)]
+    [RestrictProviderTypes(ProviderType.TLevels)]
+    [AuthorizeTLevel]
+    public class EditTLevelController : Controller
```

Comment:
Not sure why you've split this out into two controllers, `ViewTLevelController` and `EditTLevelController`?

Reply:
FormFlow really wants every action in a journey to be in a single controller (and everything in that controller to be in the journey). Since viewing is not inside the edit journey it's been split out.